examples/threat-model/threat-model.md

```diff
@@ -0,0 +1,671 @@
+
+
+## Attack Trees
+
+Attack trees offer a methodical way of describing the security of systems, based on varying attack patterns. It's important to approach the review of attack trees from a top-down perspective. The top node, also known as the root node, symbolises the attacker's primary objective. This goal is then broken down into subsidiary aims, each reflecting a different strategy to attain the root objective. This deconstruction persists until reaching the lowest level objectives or 'leaf nodes', which depict attacks that can be directly launched. 
+
+It is essential to note that attack trees presented here are speculative paths for potential exploitation. The Envoy Gateway project is in a continuous development cycle, and as the project evolves, new vulnerabilities may be exposed, or additional controls could be introduced. Therefore, the threats illustrated in the attack trees should be perceived as point-in-time reflections of the project’s current state at the time of writing this threat model. 
+
+### Node ID Schema
+
+Each node in the attack tree is assigned a unique identifier following the AT#-## schema. This allows easy reference to specific nodes in the attack trees throughout the threat model. The first part of the ID (AT#) signifies the attack tree number, while the second part (##) represents the node number within that tree.
+
+### Logical Operators
+
+Logical AND/OR operators are used to represent the relationship between parent and child nodes. An AND operator means that all child nodes must be achieved to satisfy the parent node. An OR operator between a parent node and its child nodes means that any of the child nodes can be achieved to satisfy the parent node.
+
+### Attack Tree Node Legend
+
+![AT Legend](./images/AT-legend.png)
+
+### AT0
+
+![AT0](./images/AT0.png)
+
+### AT1
+
+![AT1](./images/AT1.png)
+
+### AT2
+
+![AT2](./images/AT2.png)
+
+### AT3
+
+![AT3](./images/AT3.png)
+
+### AT4
+
+![AT4](./images/AT4.png)
+
+### AT5
+
+![AT5](./images/AT5.png)
+
+### AT6
+
+![AT6](./images/AT6.png)
+
+## About
```

Comment:
prefer if we moved `About`, `Threat Modeling Team` and `Reviewers` to the top to increase ethos
lets flatten out the names into a single line, to fit more of the introduction on the page

Reply:
Sure, I just did the modification.